import-export-cli/cmd/deleteAPIProduct.go

```diff
@@ -0,0 +1,169 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/go-resty/resty"
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/credentials"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+
+	"net/http"
+)
+
+var deleteAPIProductEnvironment string
+var deleteAPIProductName string
+var deleteAPIProductProvider string
+
+// DeleteAPIProduct command related usage info
+const deleteAPIProductCmdLiteral = "delete-api-product"
+const deleteAPIProductCmdShortDesc = "Delete API Product"
+const deleteAPIProductCmdLongDesc = "Delete an API Product from an environment"
+
+const deleteAPIProductCmdExamples = utils.ProjectName + ` ` + deleteAPIProductCmdLiteral + ` -n TwitterAPI -r admin -e dev
+` + utils.ProjectName + ` ` + deleteAPIProductCmdLiteral + ` -n FacebookAPI -v 2.1.0 -e production
+NOTE: Both the flags (--name (-n) and --environment (-e)) are mandatory.
+If the --provider (-r) is not specified, the logged-in user will be considered as the provider.`
+
+// TODO Introduce a version flag and mandate it when the versioning support has been implemented for API Products
+
+// DeleteAPIProductCmd represents the delete-api-product command
+var DeleteAPIProductCmd = &cobra.Command{
+	Use: deleteAPIProductCmdLiteral + " (--name <name-of-the-api> --provider <provider-of-the-api-product> --environment " +
```

Comment:
Here instead of api shall we use api-product in --name?

Reply:
Sure